enterprise/api/pagination.py

```diff
@@ -0,0 +1,46 @@
+"""
+Pagination helpers for enterprise api.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from collections import OrderedDict
+
+from rest_framework.response import Response
+
+from six.moves.urllib.parse import urlparse  # pylint: disable=import-error
+
+
+def get_paginated_response(data, request):
+    """
+    Fix pagination links in course catalog data.
```

Comment:
I'm not sure if "Fix" is the right term to use to describe the behavior of this function?